bot-tezos-swap/library/bot.js

```diff
@@ -0,0 +1,529 @@
+const config = require(`./${
+  process.env.BOT_ENV || "prod"
```

Comment:
failing to "dev" would be safer

Reply:
Then the changemakers will have to set the env, it might cause confusion for them